db/migrate/20150717135909_remove_unique_index_from_builds.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveUniqueIndexFromBuilds < ActiveRecord::Migration
+  def change
+    # Remove unique index on git_sha, users should be able to rebuild docker images against same SHA
```

Comment:
It could, but then we'd need to create different images for master, staging, each pod, and we wouldn't be able to reuse the same image for developers.  I think we're better off having a single image for each version of the code, and then inject configuration by ENV var or mounted files.